src/ubuntu/24.04/helix/Dockerfile

```diff
@@ -73,8 +74,12 @@ RUN curl -LO https://packages.microsoft.com/keys/microsoft.asc && \
     apt-get install -y libmsquic && \
     rm -rf /var/lib/apt/lists/*
 
+# Remove user (probably ubuntu) with UID 1000 to UID 2000 to avoid conflict with helixbot
+RUN user_id=$(id -un 1000) && \
+    /usr/sbin/userdel --force --remove $user_id
```

Comment:
Why is this also not done in the older versions of Ubuntu Helix Dockerfiles as well?

Reply:
b/c a user w/ UID `1000` doesn't exist prior to 24.04

Reply:
Well that explains it. 😄